Library/Formula/id3lib.rb

```diff
@@ -52,6 +53,12 @@ class Id3lib < Formula
     sha256 "a7881dc25665f284798934ba19092d1eb45ca515a34e5c473accd144aa1a215a"
   end
 
+  # fixes Unicode display problem in easytag: see Homebrew/homebrew-x11#123
+  patch do
```

Comment:
Has this already been applied upstream? If so you'll need to tuck this inside `stable do` to stop `head` build conflicts.